.github/workflows/python-publish.yml

```diff
@@ -31,3 +29,14 @@ jobs:
       uses: pypa/gh-action-pypi-publish@master
       with:
         password: ${{ secrets.PYPI_PASSWORD }}
+
+  dispatch:
```

Comment:
More of a question, what does dispatch as an event mean?
When is it triggered?

Reply:
mlcommons_box triggers runners build